### ISSUE TITLE
Link re-writing breaking social ctas

### ISSUE THREAD

**Issue by DavidWells** [inboundnow/cta#136]
Re: http://support.inboundnow.com/support/topic/cta-not-working/#lightbox/0/

http://www.gladstonemorgan.com/latest_news/how-to-get-the-best-results-from-offshore-savings-plans/

We need to disable the link url re-writing for these CTAs or delete the social CTAs all together.

I'm cool with both options. If we kill the CTAs we will need to migrate them out into /uploads/ like we did with that old landing page template

**Comment by DavidWells** [comment]
Link rewriting is breaking the jquery:

```
Uncaught Error: Syntax error, unrecognized expression: http://wpdev.dev/inbound/d7ooon4LILink
```

Need to disable it. How is this doable?

In fact, the link rewriting system seems to be causing a bunch of folks issues. My old version worked without rewriting URLs but by adding query params. Why did we change it?

**Comment by atwellpub** [comment]
We changed it for a couple of reaons:
* The long URLs are not as aesthetically pleasing as condenced URLs.
* Customers were asking for a better looking tracked link and the tech/methods are available.
* We also changed it to standardize the way we create tracked links across CTA, Mailer, and Leads API. 

Historical issues circling the shortened links:
* Users with default permalinks set experience issues (but this is the case for Landing Pages as well).
* Select themes where the query_var system was not picking up on the custom permalink structure, backup code has been  in place they are just not released to the public yet. Every 404 instance via customer service support request has been solved through manually implementing the latest updates. When we release we should see them disappear all together.

Updates on the way to fix all social templates.
